docs/dev-setup-non-vagrant.md

```diff
@@ -342,7 +342,7 @@ fi
 ./tools/do-destroy-rebuild-test-database
 ./manage.py compilemessages
 sudo ./scripts/lib/install-node
```

Comment:
Oh, maybe I'm wrong -- is this installed now in `node_cache.py`?  I think it'd be better to call it from `install_node`, just to keep similar things together.